test/e2e/framework/kcp.go

```diff
@@ -39,6 +42,7 @@ import (
 	"github.com/kcp-dev/logicalcluster/v3"
 	"github.com/spf13/pflag"
 	"github.com/stretchr/testify/require"
+	gopkgyaml "gopkg.in/yaml.v2"
```

Comment:
Why this one?

Reply:
hm, this is probably my IDE going crazy 🤔

Reply:
oh we need this one as the k8s helpers don't expose a decoder, bumped to v3 though.